src/screens/main/contracts/Contract.js

```diff
@@ -0,0 +1,310 @@
+import React, { Component } from 'react';
+import { View, Text, StyleSheet, ScrollView, TouchableHighlight, Dimensions, TouchableWithoutFeedback, Keyboard, SafeAreaView } from 'react-native';
+import { NavigationActions } from 'react-navigation';
+import { Icon, Button, FormLabel, FormInput, FormValidationMessage } from 'react-native-elements';
+import { connect } from 'react-redux';
+import BackWithMenuNav from '../../../components/customPageNavs/BackWithMenuNav';
+import BoxShadowCard from '../../../components/ShadowCards/BoxShadowCard';
+import LinearButton from '../../../components/LinearGradient/LinearButton';
+import RF from 'react-native-responsive-fontsize';
+import processContractByAddress from '../../../scripts/contracts/contractHelper';
+import ClearButton from '../../../components/LinearGradient/ClearButton';
+
+/**
+ * Screen is used to display the passphrase (mnemonic) of the wallet
+ */
+class Contract extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      contractLoaded: false,
+      address: '',
+      hardCodedAddress: '0x311f71389e3DE68f7B2097Ad02c6aD7B2dDE4C71',
+      wallet: this.props.wallets[0].hdWallet,
+      contractEvents: null,
+      contractFunctions: null,
+      functions: [],
+    };
+  }
+
+  async getContract() {
+    this.setState({contractFunctions: null});
+    const { contractFunctions, contractEvents } = await processContractByAddress(this.state.wallet, this.state.hardCodedAddress);
+    this.setState({contractEvents, contractFunctions });
+  }
+
+  /**
+   * Needs to be implemented
+   */
+  processFunctionInput(x, inputName, inputType) {}
+
+  /**
+   * Needs to be implemented
+   */
+  executeContractFunction() {}
+
+  /**
+   * 
+   */
+   parseFunctions() {
```

Comment:
if you're going to use fat arrow use fat arrow everywhere :)

Reply:
@GregTheGreek done